addons/crm/models/crm_lead.py

```diff
@@ -1231,13 +1231,17 @@ def get_empty_list_help(self, help):
             help_title = _('Create a new lead')
         else:
             help_title = _('Create an opportunity to start playing with your pipeline.')
-        alias_record = self.env['mail.alias'].search([
-            ('alias_name', '!=', False),
-            ('alias_name', '!=', ''),
-            ('alias_model_id.model', '=', 'crm.lead'),
-            ('alias_parent_model_id.model', '=', 'crm.team'),
-            ('alias_force_thread_id', '=', False)
-        ], limit=1)
+        alias_domain = [
+            ('crm_team_id.active', '=', True),
+            ('crm_team_id.alias_id.alias_name', '!=', False),
+            ('crm_team_id.alias_id.alias_name', '!=', ''),
+            ('crm_team_id.alias_id.alias_defaults', 'like', 'lead'),
```

Comment:
I added this condition in order to make sure that the alias is for leads.
By the way the old condition didn't seem to work as it showed an alias that was only for opportunities.